### ISSUE TITLE
getHtml() not working with cellFormatter that uses cell.getData()

### ISSUE THREAD

**Issue by Remph** [olifolkerd/tabulator#2249]
When I try to call the getHtml function I get an error in my custom cellFormatter, because it is not able to resolve the function getData() from the cell component.

**Tabulator Info**
- 4.3.0
```
var table = new Tabulator("#htmlTable",{
data:tabledata, printAsHtml:true, columnCalcs:"both", groupStartOpen:false, groupClosedShowCalcs:true, layout:"fitColumns", locale:true, movableColumns:false, resizableColumns:true, selectable:true, columns:[
{title:"beschreib", field:"beschreib", headerFilter:"input", headerSort:false, titleFormatter:menuTitleFormatter, formatter: cellFormatter},

{title:"kennzbezahlt", field:"kennzbezahlt", headerFilter:"input", headerSort:false, titleFormatter:menuTitleFormatter, formatter:cellFormatter},

], langs:{"de-de":{"headerFilters":{"default":"Spalte filtern...",} } }, });
```

Here is a "working" example with the customFormatter, if the button does not work just enter "table.getHtml()" into the console:
 https://codepen.io/anon/pen/voxVMg

It is weird since this worked in Version 4.2.0 without any problems.

**Comment by olifolkerd** [comment]
Hey @Remph 

Thanks for letting me know, Yes this is a complication because the getHTML function has been completely rebuilt as part of integrating it with other parts of the system to allow column calculations and grouped rows in the output.

I will look into a fix for this for the next release in a couple of weeks.

Cheers

Oli :)

**Comment by olifolkerd** [comment]
Hey @Remph 

I have pushed a fix for this to the 4.4 branch that will be released later today.

Cheers

Oli :)
